Source/Filter/Filter.cpp

```diff
@@ -174,7 +174,7 @@ void Filter::DoFilter (const Box& tbx,
         for         (int iz=0; iz < slen_local.z; ++iz){
             for     (int iy=0; iy < slen_local.y; ++iy){
                 for (int ix=0; ix < slen_local.x; ++ix){
-                    Real sss = sz[iy];
+                    Real sss = sx[ix];
```

Comment:
Is this what we want? Isn't this labelled `sz` in WarpX?

Reply:
Same here, maybe call these `i1` and `s1`. This will not always be `z` - these changes are here as a step to introduce 1D radial cylindrical and spherical coordinates.